client/lib/preferences-helper/PreferenceList.js

```diff
@@ -0,0 +1,64 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { partial } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { getAllRemotePreferences } from 'state/preferences/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+class PreferenceList extends Component {
+	render() {
+		const { preferences, translate, unsetPreference } = this.props;
+		return (
+			<div>
+				<a
+					href={ '/devdocs/client/state/preferences/README.md' }
+					title={ translate( 'Preferences' ) }
+				>
+					{ translate( 'Preferences' ) }
+				</a>
+				<Card className="preferences-helper__current-preferences">
+					{ preferences &&
+						Object.keys( preferences ).map( prefName => (
+							<div key={ prefName }>
+								<h5 className="preferences-helper__preference-header">{ prefName }</h5>
+								<span className="preferences-helper__value-label">
+									{ translate( 'Value: ' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **99** times:
`translate( 'Value' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pref-control-to-dev-menu)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).